### PR TITLE
KOGITO-3064 - Migrate kogito-cloud-operator PR check to GH actions us…

### PR DESCRIPTION
**Pull request by Kaitou786**
…ing KIND

See: https://issues.redhat.com/browse/KOGITO-3064
Signed-off-by: Tarun Khandelwal <tarkhand@redhat.com>

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by ricardozanini** [comment]
@Kaitou786 I believe we should remove OLM from KIND, let's try to keep it simple. @sutaakar do we required OLM to run BDD?

**Comment by Kaitou786** [comment]
@ricardozanini I'm not sure, @sutaakar asked me to have olm installed in the cluster. Right now whole this code is a mess, our first target is to first run tests successfully, which requires some changes in the tests as well. After that is done I'll format the code and squash the commits.

**Comment by Kaitou786** [comment]
Next TODO: Configure DNS Wildcard on the node for ingress

**Comment by sutaakar** [comment]
Yes, OLM is required as it is used to install all depending operators (Infinispan, Kafka, Prometheus, Keycloak...)

File: .github/workflows/Kogito-Operator-PR-Check.yaml [hunk 1]
```diff
@@ -77,4 +77,97 @@ jobs:
...
+        run: docker push localhost:5000/kiegroup/kogito-cloud-operator
+      - name: Setup JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ env.JDK_VERSION }}
+          java-package: jdk
+      - name: Build example images
+        run: ./hack/ci/build-image-examples-kind.sh
+      - name: Run Tests
+        run: hack/ci/build-image-examples-kind.sh
```
Comment (sutaakar) [inline .github/workflows/Kogito-Operator-PR-Check.yaml:173]:
```suggestion
        run: hack/ci/run-tests-kind.sh 
```

**Comment by ricardozanini** [comment]
> Yes, OLM is required as it is used to install all depending operators (Infinispan, Kafka, Prometheus, Keycloak...)

+1

I'd say to start small without those operators, run the basic tests that do not require them and then  open a new JIRA to add more things.

**Comment by ricardozanini** [comment]
40 minutes? Fair enough for now. :D

**Comment by Kaitou786** [comment]
No longer in consideration

**Pull request closed without merging**